wrappers/Mathematica/RefpropLink/RefpropLink.m

```diff
@@ -306,20 +312,39 @@
     SetOptions[ValueQ,Method->"Legacy"]         (*   Set option on ValueQ function     *)
 ];                                              (*   for all calls of ValueQ           *)
 
+(***************************************************************************************)
+(*               Mathematica 13.2 fix for R ("DegreesRankine") Units                   *)
+(* This is a workaround provided by Wolfram.  It appears that a change in the delta    *)
+(* temperature units causes "DegreesRankine" to be handled as a non-absolute quantity. *)
+(* For version 13.2, the legacy behavior in the code can be activated by with the      *)
+(* command below that sets R as an absolute temperature type.                          *)
+(* A permanent fix should be available in 13.3 and this workaround won't be necessary. *)
+(***************************************************************************************)
+If[ $VersionNumber == 13.2,                     (* ONLY if Mathematica version 12.2     *)
+    QuantityUnits`Private`hasNonZeroTempUnitQ["DegreesRankine"]=True;  (*  Workaround  *)
+];                                              (* Set R as absolute quantity          *)
+
+
 (***************************************************************************************)
 (*                            Package initialization Complete                          *)
 (***************************************************************************************)
 
+
```

Comment:
Can you please remove all the added whitespace in this file?

Reply:
Done.  Whitespace removed using VS Code.

Since the plan is to only edit in Mathematica after this as the IDE with error checking, syntax highlighting, etc., these blank lines will eventually need be added back in, but we can do that with a separate, stand-alone PR where we're not trying to identify changes to the files.

Re-ran basic tests and all is functioning correctly with no errors.